src/setupTests.js

```diff
@@ -1,6 +1,9 @@
 /** Used in jest.config.js */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Please fix lint errors in this fix with answers [here](https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies)